app/pods/components/agenda/agenda-overview/agenda-overview-item/template.hbs

```diff
@@ -124,7 +124,7 @@
                 <div>
                   {{#if @agendaitem.explanation}}
                     <span class="vl-u-text--bold">
-                      {{ t "remark-title" }}:
+                      {{t "remark-title"}}:
```

Comment:
Ik zou denken dat de linter hier niet tegen kan omdat de `:` een bare-string is.

Reply:
Blijkbaar niet. Er zijn een paar plaatsen waar dit staat. Moet ik dit in de string opnemen?

Reply:
Ik zou het wel doen ja.. misschien dat hij de : niet als een string aanziet maar dat lijkt me niet de bedoeling.. liefst wel opnemen in de string... moest iemand ooit denken dat die : gewoon een leftover was.. dan is hij ineens weg.

Reply:
Dan gaan er op andere plaatsen waar {{ t "remark-title" }}: staat ineens 2 trema's staan.

Reply:
Dit moeten we apart oplossen want dezelfde var wordt gebruikt met en zonder dubbelpunt derachter.

Reply:
Hm.. anders laat je het staan zoals het is maar moeten we een ticketje aanmaken dat dit even opgekuist wordt, mini story 0.5 punt dat er in een latere sprint snel door kan.